test-butler-app-physical-devices/build.gradle

```diff
@@ -61,15 +61,17 @@ task javadocJar(type: Jar, dependsOn: javadoc) {
     from javadoc.destinationDir
 }
 publishing {
+    def artifactFile = android.applicationVariants.find { true }.outputs.find { true }.outputFile
```

Comment:
Does this guarantee that it will always be the release variant, or do we need to do some kind of filtering here?

Reply:
Filtering on release variant, good catch.